test/test_cpu.c

```diff
@@ -27,10 +28,22 @@ void test_CPU_should_init() {
   free(c);
 }
 
+void test_CPU_load8() {
+  cpu* c = mgec_new_cpu();
+  u8 value = 242;
+  cycles cyc = load(c, XL, value);
+  TEST_ASSERT_EQUAL_INT8(242, c->xl);
+  TEST_ASSERT_EQUAL_size_t(1, cyc);
+  // Maybe?
+  TEST_ASSERT_EQUAL_INT8(0x00, c->xh);
+  TEST_ASSERT_EQUAL(0, c->sr_z);
```

Comment:
When we do set the negative flag, we should be testing for it here. Since you're using 242, it would be a negative number in two's complement.